src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/ActionPostHearingApplication.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static java.util.Objects.isNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ActionPostHearingApplication {
+    @JsonProperty("actionPostHearingApplicationTypeSelected")
+    private ActionPostHearingTypes typeSelected;
+    @Getter(AccessLevel.NONE)
+    @JsonProperty("actionSetAside")

```

Comment:
I thought JsonProperties default to field names already?

Reply:
It does normally, for some reason it was not working on these jsonunwrapped classes, it could be because of @Getter(AccessLevel.NONE)